hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/ReadBuffer.java

```diff
@@ -53,6 +55,14 @@ public void setStream(AbfsInputStream stream) {
     this.stream = stream;
   }
 
+  public void setTracingContext(TracingContext tracingContext) {
+    this.tracingContext = new TracingContext(tracingContext);
```

Comment:
In last discussion, we said new TracingContext will be created in AbfsInputStream while queueing request. Why is new needed here ?

Reply:
not needed, removed